static/base.css

```diff
@@ -429,6 +431,48 @@ color: #00bb00;}
     font-size: 17px;
 }
 
+#inspector .inspectorContents .inspectorField {
+    padding:0px;
+}
+
+#inspector .inspectorContents .inspectorField span {
+    padding:1px 3px;
+    background: #e9ecef;
+    color: #2e3236;
+    max-width: 40%;
```

Comment:
This is the max-width that seems to cause the issue for me. It makes the span's parent grow a little larger, and something blue appears underneath on the right hand side. See screenshots in main MR conversation.